test/basic_overpass_query.jl

```diff
@@ -29,14 +31,20 @@ out;
 """
 
 
-# response = HTTP.post(url, nothing, query)
-# j = JSON3.read(response.body)
-# es = j.elements
-# @info length(es)
-# fn = _data("mapcomplete_toilets.json")
-# jt = JSON3.read(read(fn))
-# ts = jt.tags
+response = HTTP.post(url, nothing, query)
+j = JSON3.read(response.body)
+es = j.elements
+@info length(es)
+fn = _data("mapcomplete_toilets.json")
+jt = JSON3.read(read(fn))
+ts = jt.tags
 
+# println.("node(around:1000, $lat, $lon)" .* node_string.(ts))
 
+# try this not haversine
+# curl 'http://router.project-osrm.org/route/v1/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219?overview=false'
 
-# println.("node(around:1000, $lat, $lon)" .* node_string.(ts))
+route_url = "http://router.project-osrm.org/table/v1/foot/-71.105650,42.363740;-71.106120,42.358030"
+response = HTTP.post(route_url)
+j = JSON3.read(response.body)
+j.durations[1][2] #this is the time (in seconds) it takes to go from the first lat/long pair to the second lat/long pair
```

Comment:
since we have a hardcoded query url, we should expect this to be relatively stable.

I would add something like 
```julia
@test j.durations[1][2] == 50
```
or if that ends up failing stochastically 
```julia
@test isapprox(j.durations[1][2], 50;atol=something)
```
testing that the json returned 50 ± something seconds